infobip_channels/sms/models/body/core.py

```diff
@@ -0,0 +1,130 @@
+from datetime import datetime
+from enum import Enum
+from typing import List, Optional, Union
+
+from pydantic import (
+    AnyHttpUrl,
+    Field,
+    StrictBool,
+    conint,
+    conlist,
+    constr,
+    root_validator,
+    validator,
+)
+
+from infobip_channels.core.models import CamelCaseModel, DateTimeValidator
+
+MINIMUM_DELIVERY_WINDOW_MINUTES = 60
+
+
+class ContentTypeEnum(str, Enum):
+    APPLICATION_JSON = "application/json"
+    APPLICATION_XML = "application/xml"
+
+
+class TrackEnum(str, Enum):
+    SMS = "SMS"
+    URL = "URL"
+
+
+class TypeEnum(str, Enum):
+    ONE_TIME_PIN = "ONE_TIME_PIN"
+    SOCIAL_INVITES = "SOCIAL_INVITES"
+
+
+class TimeUnitEnum(str, Enum):
+    MINUTE = "MINUTE"
+    HOUR = "HOUR"
+    DAY = "DAY"
+
+
+class DaysEnum(str, Enum):
```

Comment:
We have this exact enum in `mms` channel also, might be good to move it into a `core` models file and reuse it.

Reply:
done